RADAR-base/radar-prmt-android
# Feature auto update notifications

Add auto-update checking for self-release version of pRMT.

- Check the latest version of the released apk from Github
- Show a notification if a new version is available
- Check packageName and signature
- Download and install the new version
